doc/sphinx-guides/source/developers/big-data-support.rst

```diff
@@ -18,7 +40,7 @@ Install a DCM
 
 Installation instructions can be found at https://github.com/sbgrid/data-capture-module/blob/master/doc/installation.md. Note that shared storage (posix or AWS S3) between Dataverse and your DCM is required. You cannot use a DCM with Swift at this point in time.
 
-.. FIXME: Explain what ``dataverse.files.dcm-s3-bucket-name`` is for and what it has to do with ``dataverse.files.s3-bucket-name``.
+.. FIXME: Explain what ``dataverse.files.dcm-s3-bucket-name`` is for and what it has to do with ``dataverse.files.s3.bucket-name``.
```

Comment:
I'm not sure what's going on here. Maybe the JVM option was mis-documented? Also, I think `..` means it's commented out but I'm not sure.

Reply:
My change is just to update the option name to what it will be. Whether the fixme is still considered relevant, I don't know.